src/main/java/com/hladchenko/springrestapi/service/UserService.java

```diff
@@ -0,0 +1,46 @@
+package com.hladchenko.springrestapi.service;
+
+import com.hladchenko.springrestapi.entity.Note;
+import com.hladchenko.springrestapi.entity.User;
+import com.hladchenko.springrestapi.excetion.UserNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+@Service
+public class UserService {
+    private static final Map<UUID, User> userMap = new HashMap<>();
```

Comment:
I'd recomend to use ConcurrentHashMap

Reply:
Is there a case when it will break if a large number of users we'll use it at the same time?